base-transformer-class/transformer_class.py

```diff
@@ -240,13 +240,16 @@ def get_transformer_params(self, args: argparse.Namespace, metadata: list) -> di
                     # Only bother to get a timestamp if we don't have one specified
                     if timestamp is None:
                         working_timestamp = __internal__.get_first_timestamp(one_file, working_timestamp)
-        if timestamp is None and working_timestamp is not None:
+        if working_timestamp is not None:
```

Comment:
This will always assign working_timestamp while the previous code would only do that if timestamp was None. You need to preserve the functionality